source/NVDAObjects/UIA/__init__.py

```diff
@@ -712,7 +712,11 @@ def _prefetchUIACacheForPropertyIDs(self,IDs):
 			return
 		cacheRequest=UIAHandler.handler.clientObject.createCacheRequest()
 		for ID in IDs:
-			cacheRequest.addProperty(ID)
+			try:
+				cacheRequest.addProperty(ID)
+			except COMError:
+				log.debug("Couldn't add property ID %d to cache request, most likely unsupported on this version of Windows"%ID)
+				continue

```

Comment:
I think you can remove this line, as after catching this exception, the for loop will continue anyway. Thanks.